lib/uklibid/Config.uk

```diff
@@ -1,10 +1,70 @@
-config LIBUKLIBID
+menuconfig LIBUKLIBID
 	bool "uklibid: Library identifier"
-	default n
+	default y
 	# FIXME: We actually depend on <uk/assert.h> but lib/ukdebug also
```

Comment:
At the moment, `ukdebug` is already a dependency that is always on (e.g., selected by every platform). As part of plat re-arch we should get rid of this hard dependency by providing stubs when the library is off. This would then also apply for `uklibid` and we can build it without `ukdebug`.

Reply:
Overall, yes, we were saying the same thing 👍🏻